src/main/kotlin/no/nav/syfo/personoppgavehendelse/kafka/KPersonoppgavehendelse.kt

```diff
@@ -4,10 +4,5 @@ import no.nav.syfo.personstatus.domain.OversikthendelseType
 
 data class KPersonoppgavehendelse(
     val personident: String,
-    val hendelsetype: String,
+    val hendelsetype: OversikthendelseType,
```

Comment:
Vi lurer litt på om deserializeren fikser dette selv, uten at vi trenger å ta den som en string først. Må testes, ettersom at dette ikke kan testes i koden (koden kompilerer ikke hvis vi sender inn en ukjent verdi for `hendelsetype` i en test).

Reply:
Tror det skal gå bra 👍